Makefile

```diff
@@ -18,6 +18,9 @@ test: ## Run the tests
 run: ## Run the docs site locally
 	@pnpm dev
 
+docs: ## Generate documentation for a server (interactive)
```

Comment:
I think make toolkit-docs would be less confusing (given that the repo itself is named docs)